.github/workflows/test.yml

```diff
@@ -108,7 +114,24 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 1
+    - name: OSXCross for CGO Support
```

Comment:
Not sure about legal status of this.
Better not to use it, probably.

Reply:
We can just compile it on macOS instances of GHA

Reply:
> Not sure about legal status of this.
> Better not to use it, probably.

indeed..

> We can just compile it on macOS instances of GHA

I see.
if we don't have to publish the binaries via `make artifacts` over cross compiling on a Linux host, it should be okay.

I will seek a way to handle this.